src/sentry/south_migrations/0506_auto__del_field_event_data.py

```diff
@@ -0,0 +1,1467 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = True
+
+    def forwards(self, orm):
+        # Deleting field 'Event.data'
+        db.delete_column('sentry_message', 'data')
```

Comment:
Won't deleting the column happen before the code is rolled out causing problems?